sphinx/domains/index.py

```diff
@@ -68,12 +69,19 @@ class IndexDirective(SphinxDirective):
     required_arguments = 1
     optional_arguments = 0
     final_argument_whitespace = True
-    option_spec = {}  # type: Dict
+    option_spec = {
+        'name': directives.unchanged,
+    }  # type: Dict
```

Comment:
I guess mypy does not warn this line even if no type hints. Could you try that please?

Reply:
I removed the type hint here